pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -104,7 +104,7 @@ func createDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 
 			// TODO: (seba) Need to change this if multiple interface can be connected to the same network
 			// append the ports from all the interfaces connected to the same network
-			err := createSlirpNetwork(iface, *net, domain)
+			err := createSlirpNetwork(iface, *net, domain, vmi.Spec.Subdomain)
```

Comment:
Since there is no e2e test testing the fix for slirp. I wouldn't include slirp fix in this PR (I don't think it worth the time adding the test).